src/coreclr/vm/threadsuspend.cpp

```diff
@@ -5759,7 +5784,7 @@ BOOL CheckActivationSafePoint(SIZE_T ip)
 //       address to take the thread to the appropriate stub (based on the return
 //       type of the method) which will then handle preparing the thread for GC.
 //
-void HandleSuspensionForInterruptedThread(CONTEXT *interruptedContext)
+void HandleSuspensionForInterruptedThread(CONTEXT *interruptedContext, bool suspendForDebugger)
```

Comment:
Looking at this and the call site, it might be nicer to pass in the ActivationReason instead of bool.